user/timepix3/module/src/Timepix3Producer.cxx

```diff
@@ -375,12 +375,15 @@ void Timepix3Producer::DoConfigure() {
   if (!spidrctrl) {
     EUDAQ_THROW("DoConfigure: spidrctrl was not initialized. Can not configure it.");
     return;
-  }
-
+  
+}
```

Comment:
Is this only the GitHub rendering or is this indentation incorrect?

Reply:
It was an accident!